packages/dds/tree/src/simple-tree/proxies.ts

```diff
@@ -142,37 +189,44 @@ export function createObjectProxy<TSchema extends FlexObjectNodeSchema>(
 	// TODO: Although the target is an object literal, it's still worthwhile to try experimenting with
 	// a dispatch object to see if it improves performance.
 	const proxy = new Proxy(targetObject, {
-		get(target, key): unknown {
-			const field = getFlexNode(proxy).tryGetField(key as FieldKey);
+		get(target, viewKey): unknown {
+			const flexKey = getFlexKeyFromCache(viewKey as FieldKey, schema);
+			const field = getFlexNode(proxy).tryGetField(brand(flexKey));
```

Comment:
```suggestion
			const field = getFlexNode(proxy).tryGetField(flexKey);
```

Reply:
That doesn't work here because `viewKey` is of type `string | symbol`. I suspect we probably want to explicitly check for `symbol`s and respond in some (fail or noop?) way more generally. But since this is an existing pattern, I'm inclined not to tackle that in this PR.

Reply:
this is about flexKey not view key, and its type FieldKey

Reply:
Oh, I misread the diff. Will update.